HW12_2.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+def my_deco (func):
+    def wrap(*args, **kwargs):
+        return func(*args, **kwargs)
```

Comment:
You need to write the information to a file here and not in your function. The key point of this is that you will be able to decorate any function and have information is file without changing the function itself

Reply:
done